src/components/tabs/testConstants.ts

```diff
@@ -0,0 +1,13 @@
+export const TAB1_LABEL = "Tab 1";
```

Comment:
I think test-related files like this one should be placed inside `__tests__` folder

Reply:
This value should be removed from `testMatch` inside `jest.config.js`

`"<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",`